docs/general/manage-tokens/holding-rose-tokens/ledger-wallet.md

```diff
@@ -40,15 +47,36 @@ You may see both the _Oasis_ app and the _OasisVal_ app come up in the search re
 
 :::
 
+
+
 Once installed, the Oasis app will use the [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic seed stored on your Ledger wallet to generate the private & public key pairs for your accounts.
 
 ## Manage Your Tokens
 
 ### Using Oasis Wallets
 
-This is a simpler option since it allows you to connect to your Ledger wallet via a web browser app or a web browser extension.
+This is a simpler option since it allows you to connect to your Ledger wallet via web application or browser extension. In our example we will use Oasis Wallet - Web. For how to use our official wallets please use instructions in the [Oasis Wallets](../oasis-wallets/README.mdx) doc.
+
+1. Navigate to [wallet.oasisprotocol.org](https://wallet.oasisprotocol.org) and click _Open wallet_. Then click _Ledger_ when asked How do you want to open your wallet.
+![](<../../images/wallet/ledger/7_oasis_wallet_web_open_ledger.png>)
+
+2. In next step when clicking _Select the wallets to open_ button a pop-up window in your browser will appear where you will have to select your Ledger device and then click _Connect_.
+![](<../../images/wallet/ledger/8b_oasis_wallet_web_select_ledger_device_connect.png>)
+
+:::caution
+
+Make sure you open Oasis app on your Ledger before clicking _Select the wallets to open_ button. To do that navigate to Oasis app on Ledger device and use two buttons to open Oasis app, then confirm Pending Ledger review again with two buttons and Oasis Ready text should appear.
+
+:::
+
+3. After Connecting your Ledger to Oasis Wallet - Web a new pop-up will appear where you will choose one or multiple oasis1 wallets to be opened.
```

Comment:
I would prefer to be very precise w.r.t to wallets vs. account. Here, one or more accounts (from the same wallet) can be opened.

The Oasis Wallet - Web's UI needs to be updated to mention accounts as well.

Reply:
for reference: @matevz added https://github.com/oasisprotocol/oasis-wallet-web/pull/814